test/test_python_gen.py

```diff
@@ -1339,5 +1344,38 @@ def test_struct_union_default(self):
         assert s.u == self.ns2.BaseU.z
 
 
+# Adapted from:
+# http://code.activestate.com/recipes/306860-proleptic-gregorian-dates-and-strftime-before-1900/
+# Make sure that the day names are in order from 0001/01/01 until
+# 2000/08/01
+def test_strftime():
```

Comment:
Is this test executed even when it isn't part of the `unittest.TestCase` subclass?